.github/workflows/4-deploy-to-prod.yml

```
@@ -0,0 +1,103 @@
+name: Deploy to prod
+on:
+  push:
+    branches:
+      - 'releases/prod'
+
+jobs:
+
+  tag:
+    runs-on: ubuntu-latest
+    outputs:
+      new_tag: ${{ steps.github-tag-action.outputs.new_tag }}
+      changelog: ${{ steps.github-tag-action.outputs.changelog }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Bump and tag
+        id: github-tag-action
+        uses: mathieudutour/github-tag-action@v6.0
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          release_branches: .*
+          default_bump: patch
+
+  deploy-to-prod:
+    environment:
```

If they are specified as separate jobs, this will need `needs: tag`. However, I don't know what is gained by separating them. Why not include `Bump and tag` as a step in one single job?

```suggestion
    needs: tag
    environment:
```

---

Yeah good call - I'll fold them into the same job.